views/pages/index.ejs

```diff
@@ -1,12 +1,14 @@
-<%- include("../partials/header.ejs", {title: "Main Page"}) %>
+<%- include("../partials/header.ejs", {title: "Game Of Thrones"}) %>
 
 
-<p>This content comes from <code>/views/pages/index.ejs</code></p>
+<p>"That's what I do, I drink and I know things" - Tyrion Lannister</p>
 
 <section>
-  <p><a href="/db-test">Clicking this link will test to see if your db connection is ok</a></p>
+  <!--<p><a href="/db-test">Clicking this link will test to see if your db connection is ok</a></p>-->
```

Comment:
you deleted the function in app.js so might be good to remove any references to it in the rest of your code